### ISSUE TITLE
Where is `get_file`?

### ISSUE THREAD

**Issue by MikeChongCan** [llSourcell/How_to_make_a_text_summarizer#7]
In the file `vocabulary-embedding.ipynb` (https://github.com/llSourcell/How_to_make_a_text_summarizer/blob/master/vocabulary-embedding.ipynb), 

```python
---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
<ipython-input-21-ff6b5a000c0e> in <module>()
      9 if not os.path.exists(glove_name):
     10     path = 'glove.6B.zip'
---> 11     path = get_file(path, origin="http://nlp.stanford.edu/data/glove.6B.zip")
     12     get_ipython().system(u'unzip {datadir}/{path}')

NameError: name 'get_file' is not defined
```

**Comment by xtr33me** [comment]
I believe it is installed via iPython. You may want to check that it is installed correctly. That all said, I can guarantee that you can also import it via keras.utils.data_utils via keras or the newly included keras implementation in Tensorflow 1.2.  Hope that helps.

https://www.tensorflow.org/versions/master/api_docs/python/tf/contrib/keras/utils/get_file

**Comment by vbankov** [comment]
get_file is part of keras and can be found in keras and tf, import it using

`from keras.utils.data_utils import get_file`
